actions/common.go

```diff
@@ -63,9 +63,19 @@ var (
 		"workflow-manager": {"WorkflowManager"},
 	}
 
-	repoNames      = getRepoNames(repoToComponentNames)
+	repoNames = getRepoNames(repoToComponentNames)
+
+	// AdditionalGitRepoNames represents the repo names lacking representation
+	// in any helm chart, yet still requiring updates during each Workflow
+	// release, including changelog generation and creation of git tags.
+	AdditionalGitRepoNames = []string{"workflow", "charts"}
+
+	// AllGitRepoNames represent all GitHub repo names needing git-based updates for a release
+	AllGitRepoNames = getAllGitRepoNames(repoNames, AdditionalGitRepoNames)
```

Comment:
If I may suggest an alternative:

```
allGitRepoNames = append(repoNames, additionalGitRepoNames...)
```

This will save you from having to write a function to do this one thing.